public/docs/ts/latest/quickstart.jade

```diff
@@ -1,270 +1,505 @@
+      A class becomes an Angular component when we give it metadata.
+      Angular needs the metadata to understand how to build the view
+      and how the component interacts with other parts of the application.
+      
+      We define a component's metadata with the Angular `Component` function.
+      We access that function by importing it from the primary Angular library,`angular2/core`.
+    +makeExample('quickstart/ts/app/app.component.ts', 'import', 'app/app.component.ts (import)')(format=".")
+    :marked
+      In TypeScript we apply that function to the class as a *decorator* 
+      by prefixing it with the **@** symbol and invoking it
+      just above the component class:
+    +makeExample('quickstart/ts/app/app.component.ts', 'metadata', 'app/app.component.ts (metadata)')(format=".")
+    :marked
+      `@Component` tells Angular that this class *is an Angular component*.
+      The configuration object passed to the `@Component` method has two
+      fields, a `selector` and a `template`.
+
+      The `selector` specifies a simple CSS selector for a host HTML element named `my-app`.
+      Angular creates and displays an instance of our `AppComponent`
+      wherever it encounters a `my-app` element in the host HTML.
+      
+    .alert.is-helpful
+      :marked
+        Remember the `my-app` selector! We'll need that information when we write our `index.html`
+    :marked
+      The `template` property holds the component's companion template.
+      A template is a form of HTML that tells Angular how to render a view.
+      Our template is a single line of HTML announcing "My First Angular App".
+      
+    button(onclick='why("inside-app-component","app-component")' class='button button-small') Close
+
+
+a(id="main")
+.l-main-section
 :marked
-  The `template` property holds the component's companion template.
-  A template is a form of HTML that tells Angular how to render a view.
-  Our template is a single line of HTML announcing "My First Angular App".
-  
-  Now we need something to tell Angular to load this component.
-
-  ### Bootstrap it
+  ## Show it with *main.ts*
+  Now we need something to tell Angular to load the root component
   
   Add a new file , `main.ts`, to the `app/` folder as follows:
 +makeExample('quickstart/ts/app/main.ts', null, 'app/main.ts')(format=".")
-:marked
-  We need two things to launch the application:
-
-  1. Angular's browser `bootstrap` function
-  1. The application root component that we just wrote.
-  
-  We import both. Then we call `bootstrap`, passing in the **root component type**, 
-  `AppComponent`.
-
-.l-sub-section
-  :marked
-    Learn why we import `bootstrap` from `angular2/platform/browser`
-    and why we create a separate *main.ts* file in the [appendix below](#main).
-:marked
-  We've asked Angular to launch the app in a browser with our component at the root.
-  Where will Angular put it? 
-
+button(onclick='why("inside-main")' class='button button-small') Why
```

Comment:
This is a *really* long "WHY" segment. Also, it's unclear what "WHY" refers to. Maybe split it up into a few WHYs, and make sure that the most important stuff is always visible. For example:

At the top of the file, you import the Component class.

[WHY]

Next, you create a class that has an @Component decorator.

[WHY]

The @Component decorator specifies a selector—the name of the HTML tag you'll use (`<my-app>`).

[WHY]
...